### PR TITLE
[snapshot] Update package-registry to v0.11.0

### PR DESCRIPTION
**Pull request by ruflin**

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -8,6 +8,8 @@ github.com/elastic/go-ucfg v0.8.4-0.20200415140258-1232bd4774a6 h1:Ehbr7du4rSSEy
 github.com/elastic/go-ucfg v0.8.4-0.20200415140258-1232bd4774a6/go.mod h1:iaiY0NBIYeasNgycLyTvhJftQlQEUO2hpF+FX0JKxzo=
 github.com/elastic/package-registry v0.10.0 h1:xmXC571rU36qUxMV1lo3VYfwho0hUONL4aBfr9aKxng=
 github.com/elastic/package-registry v0.10.0/go.mod h1:oQx3Tg9ynuC6APd0o0OHud9kyPX6S6IzdJp/R4Hj1HY=
+github.com/elastic/package-registry v0.11.0 h1:3v7Dtvw68i3fzNzWqHloWra472wVN48tJEZJKJhAM+8=
+github.com/elastic/package-registry v0.11.0/go.mod h1:oQx3Tg9ynuC6APd0o0OHud9kyPX6S6IzdJp/R4Hj1HY=
```
Comment (ycombinator) [inline go.sum:12]: Run `go mod tidy` to clean up the old version's lines?

File: magefile.go [hunk 2]
```diff
@@ -125,6 +126,17 @@ func dryRunPackageRegistry() error {
...
+	config := `
+package_paths:
+- "../packages"
+`
+
+	err = ioutil.WriteFile("config.yml", []byte(config), 0644)
+	if err != nil {
+		return err
+	}
+
```
Comment (ycombinator) [inline magefile.go:139]: Why was this change needed as part of the version update?

Comment (ruflin) [reply magefile.go:139]: We got rid of the default public_dir in the registry. Should we introduce a default for package_paths?

Comment (ruflin) [reply magefile.go:139]: @ycombinator My hope is that in the future, we don't need the "-dry-run" of the registry anymore here and can just validate it with elastic-package.

Comment (ycombinator) [reply magefile.go:139]: TBH I'm not super familiar with the package registry code or the motivations behind https://github.com/elastic/package-registry/pull/632 to have a solid opinion here. I see @mtojek reviewed that PR so I'll defer to his opinion here.

Comment (ruflin) [reply magefile.go:139]: @ycombinator With public_dir and package_paths we had 2 ways of configuring the same thing, so we god rid of the legacy option.

Comment (mtojek) [reply magefile.go:139]: It looks like a static file. Is there an option to keep it versioned in Git instead of creating dynamically?

Comment (ruflin) [reply magefile.go:139]: Any preference on where to put it. The main requirement is that we also run the registry from this directory.

Comment (ruflin) [reply magefile.go:128]: @mtojek I updated the PR and created the file in `testing`.

File: magefile.go [hunk 3]
```diff
@@ -120,9 +120,10 @@ func dryRunPackageRegistry() error {
 	}
 	defer os.Chdir(currentDir)
 
-	err = os.Chdir(buildDir)
+	// Change to the testing directory to run package-registry from there as it contains the config for it.
+	err = os.Chdir("testing")
 	if err != nil {
-		return errors.Wrapf(err, "can't change directory to %s", buildDir)
+		return errors.Wrapf(err, "can't change directory to %s", "testing")
```
Comment (mtojek) [inline magefile.go:126]: nit: you can unify this. The `testing` const is mentioned in two places now.

**Review by mtojek — APPROVED** [review]
One small nit-picks, but it's LGTM.

File: magefile.go (hunk 3 above)
Comment (ruflin) [reply magefile.go]: agree, done.

**Comment by mtojek** [comment]
Hm.. CI job reported many failures.

**Comment by ruflin** [comment]
@mtojek I just realised tests here were still run against 7.9 instead of 7.10 so this might be the issue. Fixing now.
